fdbserver/SimulatedCluster.actor.cpp

```diff
@@ -53,17 +53,19 @@ const int MACHINE_REBOOT_TIME = 10;
 
 bool destructed = false;
 
+// given a std::variant T, this templated class will define type which will be a variant of all types in T plus Args...
```

Comment:
These seem generally useful, and not specific to SimulatedCluster, should they be moved to a header?